packages/tools/guiEditor/src/diagram/workbench.tsx

```diff
@@ -656,17 +657,15 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
             CoordinateHelper.ConvertToPercentage(guiControl, ["left", "top"]);
         }
         this.props.globalState.onPropertyGridUpdateRequiredObservable.notifyObservers();
-        return true;
     }
 
     onMove(evt: React.PointerEvent) {
         const pos = this.getScaledPointerPosition();
         // Move or guiNodes
         if (this._mouseStartPoint != null && !this._panning) {
-            let selected = false;
             this.props.globalState.selectedControls.forEach((element) => {
                 if (pos) {
-                    selected = this._onMove(element, new Vector2(pos.x, pos.y), this._mouseStartPoint!) || selected;
+                    this._onMove(element, new Vector2(pos.x, pos.y), this._mouseStartPoint!);

```

Comment:
a little nit-picky, but - is it possible to reuse a vector instead of creating a new one? onPointerDown or Up wouldn't matter so much, but pointermove happens quite often.

Reply:
good point!